app/models/attribute.py

```diff
@@ -0,0 +1,27 @@
+from .. import db
+
+user_tag_associations = db.Table(
+    'users_association', db.Model.metadata,
+    db.Column('tag_id', db.Integer, db.ForeignKey('tags.id')),
+    db.Column('user_id', db.Integer, db.ForeignKey('users.id'))
+)
+
+resource_tag_associations = db.Table(
+    'resources_association', db.Model.metadata,
```

Comment:
same thing as above here `'resources_association'` to `'resource_tag_associations'`